### PR TITLE
Vote from email

### PR DESCRIPTION
**Pull request by gdpelican**
Woowhee this was a fun one :bikini:

### CODE REVIEW

**Comment by gdpelican** [comment]
Erg, there's a bunch of noise in here from the AbilityService branch.. I'll have a go at getting that 100% ready for merge so these are easier to look at.

**Comment by robguthrie** [comment]
ok usa!

File: lineman/app/services/lmo_url_service.coffee [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+    group: (g, params) ->
+      "/g/#{g.key}/#{@stub(g.fullName())}#{@queryStringFor(params)}"
+
+    discussion: (d, params = {}) ->
+      "/d/#{d.key}/#{@stub(d.title)}#{@queryStringFor(params)}"
 
-    group: (g) ->
-      "/g/#{g.key}/#{@stub(g.fullName())}"
+    proposal: (p, params = {}) ->
+      "/m/#{p.key}/#{@stub(p.name)}#{@queryStringFor(params)}"
```
Comment (robguthrie) [inline lineman/app/services/lmo_url_service.coffee]: We're just going to link to the thread (ie: /d) soon eh? Is there any reason in your mind why we're hanging onto /m at this stage?

**Comment by robguthrie** [comment]
Looking good Jimmy!

**Pull request merged**